src/plugins/packages/themes.js

```diff
@@ -46,7 +46,6 @@ function makeLocalThemeObject(css) {
 	const themeName = css.slice(0, -4);
 	return {
 		displayName: themeName.charAt(0).toUpperCase() + themeName.slice(1),
-		filename: path.join(__dirname, "..", "..", "public", "themes", `${themeName}.css`),
```

Comment:
This is effectively never used, and having it gave a false sense that the [`/themes/:theme.css`](https://github.com/thelounge/lounge/blob/master/src/server.js#L51-L58) route is using it.